locales/de/messages.txt

```diff
@@ -0,0 +1,70 @@
+{
```

Comment:
Since the locale files are JSONs, I would use json extension.

Additionally, I would structure the files like this (i.e. include the tag prefix and separator regex):

```json
{
    "tag_prefix": "Image may contain: ",
    "separator_regex": ", | and ",
    "emoji_map": {
        …
    }
}
```